PLMS.BLL/ServicesImplementation/TaskService.cs

```diff
@@ -15,82 +16,63 @@ public class TaskService : ITaskService
         private readonly IUnitOfWork _unitOfWork;
 
         private readonly IMapper _mapper;
+        private readonly IRepository<Goal> _goalRepository;
+        private readonly IRepository<Task> _taskRepository;
+        private readonly IRepository<Category> _categoryRepository;
```

Comment:
remove _categoryRepository if not used or initialize it in the constructor of need to use